test/test_cases/send_erc1155_batch_from_mainnet_to_schain.py

```diff
@@ -46,13 +46,14 @@ def _prepare(self):
         signed_txn = self.blockchain.web3_mainnet.eth.account.signTransaction(mint_txn,
                                                                               private_key=self.config.mainnet_key)
         self.blockchain.web3_mainnet.eth.sendRawTransaction(signed_txn.rawTransaction)
+        sleep( 10 )
         self.blockchain.disableWhitelistERC1155(self.config.mainnet_key, self.config.schain_name)
+        sleep( 10 )
         self.blockchain.enableAutomaticDeployERC1155(self.config.schain_key, "Mainnet")
+        sleep( 10 )
```

Comment:
Why you added a lot of sleep function - it will increase tests execution In couple times - or do I miss something?

Reply:
Elvis knows better). We wait for TX write down into DB. That's all we do.

Reply:
These changes were performed in current PR

Reply:
will switch to `sleep( 5 )`, it should be enough